webcurator-core/src/main/java/org/webcurator/core/screenshot/ScreenshotGenerator.java

```diff
@@ -328,27 +328,27 @@ private String getScreenshotToolName() {
 
     public boolean checkIndexState(ScreenshotIdentifierCommand identifiers) throws DigitalAssetStoreException {
         File directory = new File(baseDir, identifiers.getTiOid() + File.separator + identifiers.getHarvestNumber());
-        List<SeedHistoryDTO> seedWithTimestamp = ScreenshotTimestampExtractor.getSeedWithTimestamps(identifiers.getSeeds(), directory);
+        List<SeedHistoryDTO> seedWithTimestamp = ScreenshotTimestampExtractorFromCDX.getSeedWithTimestamps(identifiers.getSeeds(), directory);
```

Comment:
Note that this will throw an exception if you set the config option cdxIndexer.enabled to 'false'. Maybe we should remove this option and simply always generate CDX indexes?

Reply:
For now I've documented this dependency in the Harvest Screenshot Configuration Guide.